client/src/components/InfoCard.tsx

```diff
@@ -1,77 +1,90 @@
+}
+const InfoCard: React.FC<props> = (props) => {
+  const toggleCommentForm = () => {
+    console.log(document.getElementById("textArea")!.style);
+    if (document.getElementById("textArea")!.style.display == "block") {
+      document.getElementById("textArea")!.style.display = "none";
+      return;
+    }
+    if (document.getElementById("textArea")!.style.display == "") {
+      document.getElementById("textArea")!.style.display = "block";
+      return;
+    }
+    document.getElementById("textArea")!.style.display = "block";
   };
 
   return (
-    <Box maxW="8xl" borderWidth="1px" borderRadius="lg" overflow="hidden">
+    <Box
+      maxW="8xl"
+      borderWidth="1px"
+      borderRadius="lg"
+      overflow="hidden"
+      display="flex"
+      flexDirection="column"
+    >
       <Box p="6">
         <Box display="flex" alignItems="baseline">
           <Badge borderRadius="full" px="2" colorScheme="teal">
             New
           </Badge>
-          <Box
-            color="gray.500"
-            fontWeight="semibold"
-            letterSpacing="wide"
-            fontSize="xs"
-            textTransform="uppercase"
-            ml="2"
-          >
-            {property.beds} beds &bull; {property.baths} baths
-          </Box>
         </Box>
 
         <Box w="100%" mt="3" fontWeight="semibold" as="h1" lineHeight="tight" noOfLines={1}>
-          {property.title}
-        </Box>
-
-        <Box>
-          {property.formattedPrice}
-          <Box as="span" color="gray.600" fontSize="sm">
-            / mo
-          </Box>
+          {props.post.address}
         </Box>
 
         <Box display="flex" mt="2" alignItems="center">
           {Array(5)
             .fill("")
             .map((_, i) => (
-              <StarIcon key={i} color={i < property.rating ? "teal.500" : "gray.300"} />
+              <StarIcon key={i} color={i < props.post.rating ? "teal.500" : "gray.300"} />
             ))}
-          <Box as="span" ml="2" color="gray.600" fontSize="sm">
-            {property.reviewCount} reviews
-          </Box>
         </Box>
       </Box>
-      <div>
-        <img src={property.imageUrl} alt={property.imageAlt} />
+      <div id="posts">
+        <Box>{props.post.caption}</Box>
+
+        <Image
+          boxSize="sm"
+          maxH="s"
+          src={props.post.imageUrl}
+          alt={"property.imageAlt"}
+          borderRadius="md"
+        />
       </div>
+
+      <Box display="flex">
+        <Button flex="1" variant="ghost" leftIcon={<BiLike />}>
+          Like
+        </Button>
+        <Button flex="1" variant="ghost" leftIcon={<BiChat />} onClick={toggleCommentForm}>
+          Comment
+        </Button>
+      </Box>
+      <form onSubmit={props.updateComments}>
+        <Box display="none" id="textArea">
+          <Textarea
+            placeholder="Add a comment"
+            name="myName"
+            onChange={(e) => props.setComment({ comment: e.target.value })}
+          />
```

Comment:
Use `useRef` if you want to access the dom element, don't use 'document.getElementById`.

See https://beta.reactjs.org/reference/react/useRef#manipulating-the-dom-with-a-ref